Lib/venv/__init__.py

```diff
@@ -16,6 +16,9 @@
 CORE_VENV_DEPS = ('pip', 'setuptools')
 logger = logging.getLogger(__name__)
 
+def _venv_path(name):
+    return sysconfig.get_path(name, scheme='venv').removeprefix(sys.prefix + os.sep)
```

Comment:
`removeprefix` being necessary hints to me that this is probably not the right approach.

Reply:
I considered setting base etc. in vars to the env path, but the variables are relative. Can certainly try to do that, but it will need more changes.

Reply:
What I meant is that we having to fetch the path from other place, and `sysconfig` of all :hushed:, and having to process it probably indicates that this approach is not great. I think this is a bit fragile, and maybe I am being too pessimistic, but I think it will break.

Reply:
OK. I've removed `removeprefix` anyway. I consider the current way less fragile than hardcoding some paths instead, but let's see what others think.